config/deploy/local/manager_config_patch.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        imagePullPolicy: Never
```

Comment:
Yes. I assume that the local deploy will be built for the `make deploy` target, and everything will run in the kind clusters. If there's a reason to assume otherwise I'm happy to change it